docs/integrations/cloudwatch/cloudwatch-logs.rst

```diff
@@ -0,0 +1,147 @@
+
+.. image:: /images/integrations/cloudwatch-overview-integrations.png
+   :alt: Screenshot of system integrations including AWS CloudWatch Logs
+
+5. Pick the endpoint by the **Endpoint name** you created earlier from the dropdown and choose **Enable**.
+
+6. Visit your AWS account and look under "CloudWatch" and explore the **Logs** section to see the data flowing within a few minutes.
+
+.. seealso:: Learn more about :doc:`/docs/integrations/cloudwatch/index`.
+
+Send logs to AWS CloudWatch from Aiven client
+---------------------------------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+For sending the logs from your Aiven service to AWS CloudWatch using the Aiven client tool, you will need:
+
+* An Aiven account with a service running.
+
+* Aiven client installed.
+
+* An AWS account, and which region it is in. 
+
+* An AWS Access Key and Secret Key. Generate the credentials by visiting **IAM dashboard** then click in **Users**, open the **Security credentials** tab, and choose **Create access key**. Click on **Download** as you will need this shortly. 
+
+.. important::
+   
+   Your AWS credentials should have appropriate access rights. According to the offical AWS documentation, the access rights required for the credentials are:
+
+   * "logs:DescribeLogStreams" which lists the log streams for the specified log group endpoint.
+   * "logs:CreateLogGroup" which creates a log group with the specified name endpoint.
+   * "logs:CreateLogStream" which creates a log stream for the specified log group.
+   * "logs:PutLogEvents" which uploads a batch of log events to the specified log stream.
+
+   .. seealso:: Find more information about `CloudWatch API <https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_Operations.html>`_.
+ 
+Configure the integration
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Open the Aiven client, and log in::
+
+    avn user login <you@example.com> --token
+
+.. seealso:: Learn more about  :doc:`/docs/tools/cli/user/user-access-token`
+
+2. Collect the following information for the creation of the endpoint between your Aiven account and AWS CloudWatch.
+
+* ``PROJECT``, the Aiven project where your endpoint will be saved to.
+* ``LOG_GROUP_NAME``, to group your log streams on AWS CloudWatch. It is an optional field. If the value is not provided, it'll be generated for you.
+* ``AWS_REGION``, the AWS region of your account.
+* ``AWS_ACCESS_KEY_ID``, your AWS access key ID.
+* ``AWS_SECRET_ACCESS_KEY``, your AWS secret access key.
+* ``ENDPOINT_NAME``, reference name for this log integration when linking it to other Aiven services.
+  
+3. Create the endpoint between your Aiven account and AWS CloudWatch.
+
+.. code:: bash
+
+   avn service integration-endpoint-create --project PROJECT \
+      -d ENDPOINT_NAME -t external_aws_cloudwatch_logs \
+      -c log_group_name=LOG_GROUP_NAME \
+      -c access_key=AWS_ACCESS_KEY\
+      -c secret_key=AWS_SECRET_ACCESS_KEY \
+      -c region=AWS_REGION
```

Comment:
I was not able to find in the `avn service types -v` those customizations options from the Aiven CLI itself. They are mentioned but not listed anywhere. Therefore, I am thinking to keep this section with a step by step how to do this from the Aiven CLI.